django/applications/catmaid/control/classification.py

```diff
@@ -279,6 +281,7 @@ def create_new_classification( workspace_pid, project_id ):
     which has an 'is_a' relation to the class 'classification_root'.
     Such a new class instance will live in the dummy project -1.
     """
+    # FIXME: Unused
```

Comment:
Can be removed. Quite a few parts of the `classification` module haven't been used over the last years / with recent CATMAID versions, so you might stumble over other unused code or bugs in here.